clusters/app.ci/supplemental-ci-images/qci-pruner.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: List
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: qci-pruner
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: qci-pruner
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: qci-pruner:latest
+    source:
+      dockerfile: |
+        FROM registry.access.redhat.com/ubi9/python-312:latest
+        RUN pip install python-dxf==11.4.0 && \
```

Comment:
python-dxf is no longer a dep

Reply:
can we use `ubi9/ubi-minimal:latest`  ?

Reply:
> python-dxf is no longer a dep

Could do you a freeze and update https://github.com/jupierce/testplatform-qci-registry-pruner/blob/main/requirements.txt accordingly. 
I cannot really see the requirements here from the code.

Reply:
> can we use `ubi9/ubi-minimal:latest` ?

We need python for this this as @jupierce implements the pruner with it in https://github.com/openshift/release/pull/53182.

Reply:
Tried the `imports` in the base image.
It seems there is no dependencies at all.